### PR TITLE
fix centos test incorrectly running ubuntu

### PR DESCRIPTION
**Pull request by saada**
* Found out the integration tests for centos were not really running centos and were instead running against Ubuntu VMs
* Even though `print_vars` showed the right values, it was only local bash values. When terraform runs, it has no clue about the correct values because it relies on global env vars. Hence the need to add `export ...` statements

### CODE REVIEW

File: test/integration/bin/internal/run-integration-tests.sh [hunk 1]
```diff
@@ -162,6 +162,18 @@ function create_image() {
...
+    export TF_VAR_gcp_image="$IMAGE_NAME" # Override the default image name.
+    export SKIP_CONFIG=1                  # No need to configure the image, since already done when making the template
+}
+
+# deprecated
+#
+# Note 1: not sure what the goal of creating images was but it appears to be
+# misbehaving due to gcp not playing nicely with slashes in image names
+# Note 2: custom images are not using in any integration test path, not sure
+# what custom images were for but I'm guessing build caching?
```
Comment (bboreham) [inline test/integration/bin/internal/run-integration-tests.sh:176]: This code is originally from https://github.com/weaveworks/build-tools/tree/master/provisioning; it creates images with all the packages and images preloaded to save time in each test run.

File: .circleci/config.yml [hunk 2]
```diff
@@ -187,8 +187,8 @@ jobs:
           SRCDIR: /src/github.com/weaveworks/wksctl
           KUBECTL_URL: https://dl.k8s.io/v1.10.5/kubernetes-client-linux-amd64.tar.gz
           KUBECTL_CHECKSUM: da9d557989a0b9671a610f21642052febb8f70c3cf144c98a8a4f7ecab6bafe2
-          CREATE_IMAGE: 0
-          USE_IMAGE: 0
+          CREATE_IMAGE: 1
```
Comment (bboreham) [inline .circleci/config.yml:190]: What was this supposed to do?  You took out the call to the place it gets used.

File: test/integration/bin/internal/run-integration-tests.sh (hunk 1 above)
Comment (saada) [reply test/integration/bin/internal/run-integration-tests.sh:176]: we found out the images are not actually used by the tests

File: .circleci/config.yml (hunk 2 above)
Comment (saada) [reply .circleci/config.yml:190]: we thought we used the images created but they never got used by terraform

**Pull request merged**